### PR TITLE
Rust gerg

### PR DESCRIPTION
**Pull request by royvegard**
This implements Gerg2008 in Rust.

### CODE REVIEW

File: AGA8CODE/RUST/tests/gerg-tests.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    gerg_test.density(0);
+    gerg_test.properties();
+
+    assert!(f64::abs(gerg_test.d - 12.79828626082062) < 1.0e-4);
+    assert!(f64::abs(gerg_test.mm - 20.5427445016) < 1.0e-8);
+    assert!(f64::abs(gerg_test.p - 50000.00000000001 ) < 1.0e-8);
+    assert!(f64::abs(gerg_test.z - 1.174690666383717 ) < 1.0e-5);
+    assert!(f64::abs(gerg_test.dp_dd - 7000.694030193327 ) < 1.0e-1);
+    // TODO: This next assert will probably have to get a new reference value.
```
Comment (royvegard) [inline AGA8CODE/RUST/tests/gerg-tests.rs]: Ref. #8

**Comment by ianhbell** [comment]
Great! Are we ready to roll on this one?

**Comment by royvegard** [comment]
Checks are green - I believe we are!
I want to improve the API, but that will come i another branch.

**Comment by ianhbell** [comment]
Thanks!

**Pull request merged**